### PR TITLE
Feature/enhanced ui

### PR DESCRIPTION
**Pull request by sultanaltowerqi**
Latest version of UI.

### CODE REVIEW

File: www/face_matcher/forms.py [hunk 1]
```diff
@@ -19,3 +23,17 @@ def clean_face_source_filter(self):
...
         return data
+
+    def clean_max_results(self):
+        data = int(self.cleaned_data['max_results'])
+        if data < 1 or data > 20:
+            raise forms.ValidationError('You have to select correct count')
+        return data
+
+
+class FmUserCreationForm(UserCreationForm):
```
Comment (full-of-foo) [inline www/face_matcher/forms.py]: Perhaps rename class to `RegistrationForm`

File: www/face_matcher/views.py [hunk 2]
```diff
@@ -52,10 +63,72 @@ def faces(request):
...
+
+    context_dict = {
+        'history': history
+    }
+    return render(request, 'face_matcher/history.html', context_dict)
+
+
+@login_required
+@csrf_exempt
+def ajax_get_history(request, id):
```
Comment (full-of-foo) [inline www/face_matcher/views.py]: Renaming to `get_json_histroy` is a bit nicer

**Comment by full-of-foo** [comment]
@sultanaltowerqi can we get some tests for this work please :smile:?

**Comment by magrossi** [comment]
Hi Sultan,

Can you see the comments from Anthony in GitHub and modify your code
please? I'll merge it after the proposed changes have been made.

Thank you!

Marcelo

**Pull request merged**